server/lib/express.js

```diff
@@ -74,6 +74,14 @@ export default function(app) {
 
   app.use(multer());
 
+  app.use(require('forest-express-sequelize').init({
```

Comment:
Our linter won't be happy with this. Mind moving this to the top as an Import statement?